src/app/body/task-details/upload-files/upload-files.component.html

```diff
@@ -13,12 +13,16 @@
 ***********************************************************/ -->
 <div *ngIf="(uploadService.filesData )?.length != 0; else upload">
     <div class="row" style="max-height: 40px;">
-        <h4 class="col-md-10 col-10 pt-2 mb-0">Uploaded Files</h4>
+        <h4 class="col-sm-10 col-10 pt-2 mb-0 mr-4">Uploaded Files</h4>
```

Comment:
why using sm, not a good bootstrap that we follow, also mr is not good strategy, try something else